spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-repository-sample/README.adoc

```diff
@@ -32,19 +33,16 @@ $ export SPANNER_EMULATOR_HOST=localhost:9010
 ----
 
 === Running the application
+
 Run `$ mvn clean install` from the root directory of the project.
 
-Run the `SpannerRepositoryExample` or the `SpannerTemplateExample` Spring Boot console apps.
+Run the `SpannerRepositoryExample`  Spring Boot console apps.
 This causes the app to generate sample `Trade` objects filled with some demo data and persists them to the Cloud Spanner table created earlier.
 These POJOs are then retrieved and their contents are printed in the browser.
 
 .. To run the SpannerRepositoryExample:
```

Comment:
We don't need this line anymore, since the description above it ("Run the ... console app") only applies to one example now.